selene_sdk/samplers/multi_file_sampler.py

```diff
@@ -3,27 +3,39 @@
 FileSampler for each mode of sampling (train, test, validation).
 The MultiFileSampler is therefore a subclass of Sampler.
 """
+from torch.utils.data import DataLoader
 
 from .sampler import Sampler
-
+from .dataloader import SamplerDataLoader
 
 class MultiFileSampler(Sampler):
     """
-    This sampler contains individual file samplers for each mode.
-    The file samplers parse .bed/.mat files that correspond to
-    training, validation, and testing and MultiFileSampler calls on
-    the correct file sampler to draw samples for a given mode.
-
-    Attributes
+    This sampler draws samples from individual file samplers or data loaders 
+    that corresponds to training, validation, and testing (optional) modes. 
+    MultiFileSampler calls on the correct file sampler to draw samples 
+    for a given mode. 
+
+    MultiFileSampler can use either file samplers or data loaders for 
+    different modes. Using data loaders for some mode while using file samplers
+    for other modes are also allowed. The file samplers parse data files 
+    (e.g. bed, mat, or hdf5) to provide samples. The data loaders can provide
+    multi-worker iterators that draw samples from a sampler or a file sampler. 
+    As data loaders support parallel sampling, they are generally recommended 
+    for sampling speed.
+    
+    Parameters
     ----------
-    train_sampler : selene_sdk.samplers.file_samplers.FileSampler
+    train_sampler : selene_sdk.samplers.file_samplers.FileSampler or
```

Comment:
It may take a bit of time for user to understand the relative import though - I feel like it could be either just the base name 'FileSampler or SamplerDataLoader' (they can look up by search) for brevity or the full name for clarity.